tests/dictful/conftest.py

```diff
@@ -53,12 +53,38 @@ async def iter_servers(self):
 
 @pytest.fixture(scope="session")
 def dict_models():
-    @DictRegistry.document("FooV1")
+    @DictRegistry.document("Foo1")
+    class AncientFoo(chaise.dictful.Document):
+        # spam is uppercase
+        pass
+
+    @DictRegistry.document("Foo2")
+    class OldFoo(chaise.dictful.Document):
+        # spam is lowercase
+        pass
+
+    @DictRegistry.migration(AncientFoo, OldFoo)
+    def foo1_migration(old):
+        return OldFoo(bar=old["bar"].lower())
+
+    @DictRegistry.document("Foo3")
     class Foo(chaise.dictful.Document):
+        # spam is titlecase
+        pass
+
+    @DictRegistry.migration(OldFoo, Foo)
```

Comment:
Best practice will clearly be `Foo` is current, and `OldFoo` becomes `FoovX` which allows rename tooling to make some of this _quite_ easy to manage long term.